guides/common/modules/con_major-project-components.adoc

```diff
@@ -0,0 +1,5 @@
+[id="Major-{Project}-Components_{context}"]
+= Major {Project} Components
+
+A typical {Project} deployment consists of the following components: a {ProjectServer}, {SmartProxyServers} that mirror content from {ProjectServer}, and managed hosts that receive content from {ProjectServer} and {SmartProxyServers}.
+Learn about these components and the features they provide.
```

Comment:
Very unlike the docs. Maybe reword to "the following chapters provide an overview of a typical/hypotential setup" or smiliar. Reads a bit marketing-y.